src/Microsoft.AspNetCore.ResponseCaching/Interfaces/IResponseCacheStore.cs

```diff
@@ -2,13 +2,14 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Threading.Tasks;
 
 namespace Microsoft.AspNetCore.ResponseCaching
 {
     public interface IResponseCacheStore
     {
-        object Get(string key);
-        void Set(string key, object entry, TimeSpan validFor);
-        void Remove(string key);
+        Task<object> GetAsync(string key);
+        Task SetAsync(string key, object entry, TimeSpan validFor);
+        Task RemoveAsync(string key);
```

Comment:
The original idea was to make use of this for invalidation. I'm not sure if that's still on the table but this would be needed if we eventually want to add it.